content/en/docs/concepts/overview/components.md

```diff
@@ -33,7 +33,7 @@ The control plane's components make global decisions about the cluster (for exam
 Control plane components can be run on any machine in the cluster. However,
 for simplicity, set up scripts typically start all control plane components on
 the same machine, and do not run user containers on this machine. See
-[Building High-Availability Clusters](/docs/admin/high-availability/) for an example multi-master-VM setup.
+[Building High-Availability Clusters](/docs/admin/high-availability/) for an example multi-control-plane-VM setup.
```

Comment:
“multi-control-plane-VM” uses lots of hyphens, and I think could be hard to understand.

We should reword from the current text. How do feel about:
```suggestion
[Creating Highly Available clusters with kubeadm](/docs/setup/production-environment/tools/kubeadm/high-availability/)
for an example control plane setup that runs across multiple VMs.
```

I also fixed the hyperlink.